swift/apple/FirezoneNetworkExtension/CallbackHandler.swift

```diff
@@ -22,20 +22,19 @@ public protocol CallbackHandlerDelegate: AnyObject {
     tunnelAddressIPv6: String,
     dnsAddresses: [String]
   )
-  func onTunnelReady()
```

Comment:
Do you now just assume that the tunnel is ready after `connect`? I think that makes sense. I guess `connlib` should fail using `on_disconnect` if we cannot initialize the device.

Reply:
We assume it's ready after the first `onSetInterfaceConfig`